### DISCUSSION TITLE
Custom Select in Mantine v5

### DISCUSSION THREAD

**Discussion by SilvaLucas92** [mantinedev/mantine#6296] [Q&A]
Hi! I'm trying to create a custom select in v5 to handle big data. I wanted to know how to create a custom select dropdown using InputBase

**Comment by rtivital** [comment]
Use Select component source code as a reference

**Reply by SilvaLucas92** [reply]
can I create a custom dropdown??

**Reply by rtivital** [reply]
In your code, you can organize logic any way your application requires. Mantine does not put any restrictions on how your components are implemented.

**Reply by SilvaLucas92** [reply]
so for example I wanted to create a dropdown with infinite scroll. Should i use dropdownComponent prop? I wanted to check the source code but I think it doesnt exist because github warned me that the page doesn't exist
